Problem -2.java

```diff
@@ -0,0 +1,134 @@
+import java.util.Scanner;
+public class Shapes {
+    public static void main(String[] args) {
+        Scanner in= new Scanner(System.in);
+        System.out.println("Select the Shape to calculate Area,Perimeter and Volume(3-D) for :" );
+        System.out.println("1. Circle :" );
+        System.out.println("2. Rectangle :" );
+        System.out.println("3. Triangle :" );
+        System.out.println("4. Square :" );
+        System.out.println("5. Parallelogram :" );
+        System.out.println("6. Cube :" );
+        System.out.println("7. Sphere :" );
+        System.out.println("8. Cylinder :" );
+        System.out.println("\n Enter your choice :");
+        switch(in.nextInt())
+        {
+            case 1 :
+                    System.out.println("Enter radius of the Circle:  " );
+                    double rad = in.nextDouble();
```

Comment:
Can you please break these cases into multiple classes and also handle the exception if user enters the non-integer value?

Reply:
Ok sir i will modify it .